### ISSUE TITLE
"Update nuget package" is retreiving unlisted versions

### ISSUE THREAD

**Issue by knocte** [mono/monodevelop#6586]
Steps to reproduce:
1. Publish some nuget package Foo, version 3.0.
2. Change the nuget package version in your sources, and publish 2.99.1 version.
3. Unlist the v3.0 from the nuget website.
4. Open VS4Mac, create a new solution/project "Bar".
5. Add a reference from Bar to Foo v.2.99.1 nuget package.
6. Upload new version of Foo 2.99.2.
7. Right click on the nuget reference -> update.

Current results: it updates to version 3.0 even if it's unlisted.
Expected results: it should update to version 2.99.2 of the package.

Tested with version: 7.6.11 (build 9)

**Comment by mrward** [comment]
I filed this against [NuGet](https://github.com/NuGet/Home/issues/4553) a while ago and that issue seems to have been closed with a fix. Seems to have been included in NuGet 4.8 which is not yet used by Visual Studio for Mac.

**Comment by mrward** [comment]
Possibly. Needs testing.

**Comment by mrward** [comment]
Mmm, do not seem to able to reproduce this with VS Mac 7.7.1. I uploaded two versions of a test NuGet package -Test.Xam.Unpublished. Version 0.2.0 is unlisted, whilst 0.1.0 is listed. 0.1.0 can be searched for and installed. Tried the various Update menus and it does not seem to find 0.2.0 and does not update.

I will try a release version instead of a pre-release version, but nuget.org takes a long time to update before these will be available.
